src/styles/index.css

```diff
@@ -0,0 +1,69 @@
+@import "./reset.css";
+@import "./main-property.css";
+@import "./stars.css";
+@import "./price.css";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 410px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+}
+
+.card__code {
+  margin-top: 4px;
+}
+
+.card__action {
+  position: relative;
+  margin-top: 16px;
+}
+
+.review {
```

Comment:
it doesn't seem to be separate block, please, convert it to element by naming it to card__reveiw